src/_scss/pages/search/results/table/_resultsTableMessageContainer.scss

```diff
@@ -0,0 +1,48 @@
+.results-table-message-container {
+    display: flex;
+    flex-flow: column wrap;
+    position: relative;
+    background-color: rgb(246, 246, 246);
```

Comment:
We typically use hex values. Not sure if theres a hex defined as a variable that you might be able to use here?

Reply:
Updated!